Articles/Blog/InclusiveLanguageCodeUpdateExample.md

```diff
@@ -0,0 +1,195 @@
+
+* *Did you have a good understanding of what was involved going in?*
+
+   > Agreeing on the replacement terminology certainly took longer than I expected.
+   > That ended up being a good thing since it prompted us to open up the conversation with several teams at Sandia and even other DOE labs.
+
+### Closing thoughts
+
+While there was [much discussion](https://www.google.com/search?q=%22master%22+%22slave%22+terminology+in+software&sxsrf=AOaemvIy5OkrC1xJEV_CJrUNIG3Tvmtlrg:1643308409486&source=lnt&tbs=qdr:y&sa=X&ved=2ahUKEwjdlYPwyNL1AhVuD0QIHXqoBOsQpwV6BAgBEBk&biw=1616&bih=948&dpr=1) of the `master/slave` issue in the media throughout 2020, it has been a [known](https://tools.ietf.org/id/draft-knodel-terminology-00.html) and [long-standing](https://www.jstor.org/stable/40061475) issue long before 2020, with the earliest online mentions going back [nearly 20 years](http://www.cnn.com/2003/TECH/ptech/11/26/master.term.reut/index.html), and other public discourse almost certainly much earlier.
+
+Beginning in 2021, less than a year after the summer of 2020, the broader US public has seen what can only be described as a [*backlash*](https://fivethirtyeight.com/features/white-backlash-is-a-type-of-racial-reckoning-too/) against these and other [DE&I efforts](https://sloanreview.mit.edu/article/fighting-backlash-to-racial-equity-efforts/).
+Hopefully, the community will resist this [habit](https://www.tandfonline.com/doi/full/10.1080/10705422.2021.1998875) and continue to practice and make progress in making HPC/CSE more inclusive.
+
+Some readers may wonder if the people spearheading the efforts described here were [BIPOC](https://www.healthline.com/health/bipoc-meaning).
+Yes, they were.
+Hopefully, the community will recognize the importance of unburdening our BIPOC colleagues from having to always be the ones doing the heavy lifting in efforts aimed at making HPC/CSE more inclusive.
+On the other hand, we must also take care that when we engage in such work, we do so with a commitment of ensuring the voices of our BIPOC colleagues are informing priorities and process.
+
+Given the current social justice climate in which we all operate, some readers may feel that inclusive language efforts are nowhere near enough to meet the moment and are really just a [distraction](https://www.wired.com/story/tech-confronts-use-labels-master-slave/) from bigger issues.
+It's hard to argue with that.
+Nonetheless, others may feel like such efforts go way too far.
+While we can acknowledge both perspectives exist, it is worth considering that when we’re [accustomed](https://www.huffpost.com/entry/when-youre-accustomed-to-privilege_b_9460662) to the status quo, being called to invest effort to be more inclusive may feel like oppression.
+
+*Prepared by LLNL under Contract DE-AC52-07NA27344 as document LLNL-AR-832434.*
```

Comment:
Does this need to be visible?  Could we turn it into a comment?

Reply:
> Does this need to be visible? Could we turn it into a comment?

Yes, it does. Fortunately, its only a single line 😉

Reply:
Okay.  I realize that we also need your bio. ~50-100 words, under a L4 heading.